react/legacy/components/ProductSummaryPrice.js

```diff
@@ -68,9 +68,9 @@ ProductSummaryPrice.propTypes = {
   /** Set installments' visibility */
   showInstallments: PropTypes.bool,
   /** Text of selling Price's label */
-  labelSellingPrice: PropTypes.string,
+  labelSellingPrice: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
   /** Text of selling Price's label */
-  labelListPrice: PropTypes.string,
+  labelListPrice: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Comment:
:wat:

Reply:
ah this is to remove proptype warnings where `labelListPrice` was set to `false` (probably is being set like `labelListPrice={foo && foo.bar}` 

it's weird but 🤷‍♂